src/components/ImportDropdown/ImportDropdown.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+
+import { useClient } from 'cozy-client'
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import ListItemIcon from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Camera from 'cozy-ui/transpiled/react/Icons/Camera'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import IconStack from 'cozy-ui/transpiled/react/IconStack'
+import FileDuotoneIcon from 'cozy-ui/transpiled/react/Icons/FileDuotone'
+
+import Konnector from '../../assets/icons/Konnectors.svg'
+import { getCssVariableValue } from 'cozy-ui/transpiled/react/utils/color'
+import { getFilteredStoreUrl } from '../../utils/getFilteredStoreUrl'
+
+const ImportDropdown = ({ label, icon }) => {
+  const { t } = useI18n()
+  const client = useClient()
+
+  const goToStore = () => {
+    window.location = getFilteredStoreUrl(client)
+  }
+
+  return (
+    <>
+      <ListItem divider={true}>
+        <IconStack
+          backgroundIcon={
+            <Icon
+              icon={FileDuotoneIcon}
+              color={getCssVariableValue('dodgerBlue')}
```

Comment:
No need to use `getCssVariableValue`. 

You can access directly the CSS variable by doing: 
color="var(--dodgerBlue)" 

BUT the case when we need to use a "non semantic" variable is pretty rare. So let's see with @joel-costa if we really want to use the dodgerBlue or if the color should be semantic.

Reply:
Vu avec @joel-costa, on va utiliser `color="var(--dodgerBlue)"`